practice/lib/practice/mid_char.ex

```diff
@@ -15,7 +16,7 @@ defmodule MidChar do
     |> Kernel.-(1)
```

Comment:
I think something like `get_middle_index` might be a more descriptive name for this function.

Reply:
Your right but Code wars test suite sometimes complains if I change the name of the top function. 
Also complains about file name changes. So I try not to change the top level functions too much

Reply:
Thank @apreifsteck

Reply:
@DoggettCK I end up writing a one-line function to incorporate you comment since I can do the `>>>` inside of a pipe. I don't know if that is better or worst. Also thanks for calling out the bitwise module